tests/test_inner.py

```diff
@@ -516,7 +511,7 @@ class Exit(ExitPlugin):
       },
      True,  # is fatal
      True,  # logs error
-    ),
+    ),  # noqa
```

Comment:
Is this for `E124 closing bracket does not match visual indentation`? Does it just need to be indented one space?